### ISSUE TITLE
Add Input Field for Custom Font Value Alongside Slider

### ISSUE THREAD

**Issue by Mefisto04** [mlc-ai/web-llm#604]
**Describe the feature**  
Currently, the font size value is adjusted solely using a drag slider. This method may not provide the best user experience, as users might prefer a more precise way to set the font size rather than relying on a slider alone.

---

**To Reproduce**  
1. Go to the font adjustment section of the application.
2. Attempt to change the font size using the drag slider.
3. Notice the difficulty in achieving a specific font size without the ability to enter a precise value.

---

**Expected behavior**  
It would be beneficial to have an input field alongside the slider. This allows users to enter a custom font size directly, providing a more flexible and user-friendly experience for adjusting font values.

---

**Screenshots**  
![image](https://github.com/user-attachments/assets/ea4d7ff4-ca6b-4361-a33b-3df247ab255c)

---

**Additional context**  
Implementing an input field alongside the slider will enhance usability, allowing users to quickly and accurately set their desired font size without the limitations of the slider. This change could improve the overall user experience significantly.

**Comment by Neet-Nestor** [comment]
Were you using it on mobile device? It's just kinda confusing to me since it seems to me that with the current range it would be easy enough for me to select any given value with the slider.

**Comment by Mefisto04** [comment]
yes, but from the point of user experience the slider doesn't make any sense if you can just take input of that particular range, it's just making it hard to slide to that number appropriate and then release it.

**Comment by Neet-Nestor** [comment]
I see, I can make this change later.

**Comment by Anonymous-RD** [comment]
interesting , i can fix that issue related to frontend , would you like to assign it to me
